src/db/json/books-dao-example.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
```

Comment:
What is this file for? Please make sure you carefully double-check all the changes before pushing anything to git.

Reply:
It was pre-added in the API skeleton when I cloned it.